src/main/java/com/ericsson/ei/handlers/RmqHandler.java

```diff
@@ -150,6 +162,33 @@ public ConnectionFactory connectionFactory() {
         return cachingConnectionFactory;
     }
 
+    @Bean
+    public RabbitManagementTemplate deleteBindingKeys() {
+    	int rabbitPort=port;
+    	if(port==5672){
+    		rabbitPort=15672;
+    	}
+        rabbitManagementTemplate = new RabbitManagementTemplate("http://"+host+":"+rabbitPort+"/api/",user,password);
```

Comment:
What happens if un-secure http is disabled and only https is available in target environment? 
Then this implementation will not work, I guess.

Reply:
> How is other Eiffel component handling restart of Eiffel components with new bindingKey ?

This binding change will happen from some of the Eiffel components, but rebind is not implemented in Eiffel components.